sauce/features/budget/display-goal-amount/settings.js

```diff
@@ -1,8 +1,13 @@
 module.exports = {
   name: 'DisplayTargetGoalAmount',
-  type: 'checkbox',
+  type: 'select',
   default: false,
```

Comment:
The default value should match the default "options" value. In this case that would be 0.